src/Scene/MobileMappingUpdater.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Generated On: 2016-11-15
+ * Class: DefaultUpdater
+ * Description: Updaters for MobileMappingLayers.
+ */
+
+function MobileMappingUpdater() {
```

Comment:
Is MobileMapping supported yet? if not I'd rather not add untested class like this one.